libsignify/src/lib.rs

```diff
@@ -6,10 +6,17 @@
 //! `libsignify` can verify and create signatures that are interoperable with BSD signify.
 //! You can read more about the ideas and concepts behind `signify` in [Securing OpenBSD From Us To You](https://www.openbsd.org/papers/bsdcan-signify.html).
 //!
+//! This crate is `#![no_std]` by default.
```

Comment:
We should document that it _does_ use `liballoc` though.

Reply:
Good idea. Added a quick note that it needs `liballoc` and a functional allocator on the platform.